packages/preview/simplebnf/0.1.0/simplebnf.typ

```diff
@@ -0,0 +1,53 @@
+#let bnf(
+  ..body,
+) = {
+  let content = body.pos().flatten()
+
+  table(
+    columns: (
+      auto,
+      auto,
+      auto,
+      auto,
+      auto,
+    ),
+    align: (
+      right,
+      center,
+      center,
+      left,
+      left,
+    ),
+    inset: 2.7pt,
+    stroke: none,
+    ..content,
+  )
+}
+
+#let Prod(
```

Comment:
Typst packages generally use kebab-case names, without capital letters. If you want to match this convention, you should rename `Prod` and `Or`. But if there is a good reason to have them capitalized (or if you just prefer it this way), you can keep it as is too.